lwt/jbuild

```diff
@@ -1,12 +1,39 @@
+(* -*- tuareg -*- *)
```

Comment:
True but the test requires it in some sense... It needs `unix, lwt.unix, io-pages-unix` to be compiled (that is why it requires `nbd.lwt` at the moment)